src/webui/api/appcontroller.cpp

```diff
@@ -187,6 +187,7 @@ void AppController::preferencesAction()
     data[u"proxy_username"_qs] = proxyConf.username;
     data[u"proxy_password"_qs] = proxyConf.password;
 
+    data[u"proxy_hostnames"_qs] = session->isProxyHostnamesEnabled();
```

Comment:
You should also increase the minor version of `API_VERSION` in webapplication.h.

Reply:
Done